### ISSUE TITLE
quick_tutorial: deform static_url call broken, & uneeded <script> tag

### ISSUE THREAD

**Issue by perryhook** [Pylons/pyramid#1857]
I'm not sure if I'm doing something wrong or misunderstand, but in version 1.5.7 of the quick_tutorial the way the `wikipage_addedit.pt` template loads deform's static resources seems broken on lines 7 and 12. 

As far as I can tell, it was broken by commit https://github.com/Pylons/pyramid/commit/5c5edadd089f3c3f13751bde3f691ad19339b551 which was a bugfix for issue #1266, the purpose of which was to (from what I can tell) fix a "mistake" by eliminating the concatenation in the calls to `static_url`. But, for me, the `reqt` doesn't include the `'deform:static/'` part, as sbivol said, so it _does_ need to be concatenated on first.

Currently lines 12 and 13 are:
```html
<link rel="stylesheet" type="text/css"
              href="${request.static_url(reqt)}"/>
```
But to work for me, I need to return it to how it was before this commit:
```html
 <link rel="stylesheet" type="text/css"
              href="${request.static_url('deform:static/' + reqt)}"/>
```
Line 7 should be corrected in the same manner.

Could there be something unique to our environments that is causing this difference? Could using pip instead of easy_install have caused this issue (I've been using pip)? I am also using Python 2.7. Could that be an issue?

Also, the extra `<script>` tag to load jQuery on lines 9 and 10 (added in commit c1066ef70a944f8cd5ecdd730662609e4c715075 ) is unneeded and does nothing, as the (corrected) `<tal:block...>` that loads the 'js' requirements in lines 11-14 takes care of loading jQuery. Yes, it loads an older version, but as it stands lines 9 and 10 do nothing anyway because jQuery is loaded again in lines 11-14.

Perhaps I'm missing something here? Can anyone else confirm these issues?

**Comment by stevepiercy** [comment]
What is the URL of the Quick Tutorial step that is causing you issues?

> Could using pip instead of easy_install have caused this issue (I've been using pip)?

The Quick Tutorial does not currently support users who go off the beaten path with pip. Yes, it's possible that is causing the issue. Suggest trying it in a clean environment with setuptools/easy_install as stated in the beginning of the QT instructions.

**Comment by perryhook** [comment]
Thank you Steve, pip was indeed the issue. I started with a clean python3.4 environment  and used easy_install and it works. Should have checked before I posted. I'm sorry I didn't realize my mistake!

I was looking at:
http://docs.pylonsproject.org/projects/pyramid/en/latest/quick_tutorial/forms.html

It seems that deform installed via pip installs doesn't work the way the same as with easy_install. When it didn't work for me I compared to the code in the "getting_started" tutorial that had worked for me previously:
http://docs.pylonsproject.org/projects/pyramid-tutorials/en/latest/getting_started/09-forms/index.html

Your added line to load jQuery is also necessary when using easy_install.

Thanks again Steve for the quick response.
